app/src/main/java/org/exoplatform/tool/ServerUtils.java

```diff
@@ -121,7 +123,11 @@ public void onResponse(Call<PlatformInfo> call, Response<PlatformInfo> response)
           @Override
           public void onFailure(Call<PlatformInfo> call, Throwable t) {
             Log.e(LOG_TAG, "Unable to retrieve platform information", t);
-            callback.onServerInvalid();
+            if (t.toString().contains("java.net.ConnectException") || t.toString().contains("java.net.UnknownHostException")){
```

Comment:
Use instanceOf instead
```suggestion
            if (t instanceOf java.net.ConnectException || t instanceOf java.net.UnknownHostException)){
```